mypy.ini

```diff
@@ -0,0 +1,24 @@
+[mypy]
+
+# Punting on the generated terrain files for the moment. The generator needs to be fixed and re-run.
+
+[mypy-dcs.terrain.caucasus]
+ignore_errors = True
+
+[mypy-dcs.terrain.marianaislands]
+ignore_errors = True
+
+[mypy-dcs.terrain.nevada]
+ignore_errors = True
+
+[mypy-dcs.terrain.normandy]
+ignore_errors = True
+
+[mypy-dcs.terrain.persiangulf]
+ignore_errors = True
+
+[mypy-dcs.terrain.syria]
+ignore_errors = True
+
+[mypy-dcs.terrain.thechannel]
+ignore_errors = True
```

Comment:
super minor, but newline is missing here at the end of file :)

Reply:
Whoops, thanks. Fixed.